tests/rule_based_profiler/data_assistant/test_volume_data_assistant.py

```diff
@@ -2448,6 +2448,108 @@ def test_get_metrics_and_expectations_using_implicit_invocation(
     )
 
 
+@freeze_time("09/26/2019 13:42:41")
+def test_get_metrics_and_expectations_using_implicit_invocation_with_exclude_column_names_directive(
+    quentin_columnar_table_multi_batch_data_context,
+    set_consistent_seed_within_numeric_metric_range_multi_batch_parameter_builder,
+    quentin_expected_metrics_by_domain,
+    quentin_expected_expectation_suite,
+    quentin_expected_rule_based_profiler_configuration,
+):
+    context: DataContext = quentin_columnar_table_multi_batch_data_context
+
+    batch_request: dict = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+    }
+
+    exclude_column_names: List[str] = [
+        "dropoff_datetime",
+        "store_and_fwd_flag",
+        "extra",
+        "congestion_surcharge",
+    ]
+    data_assistant_result: DataAssistantResult = context.assistants.volume.run(
+        batch_request=batch_request,
+        **{
+            "exclude_column_names": exclude_column_names,
+        },
+    )
```

Comment:
Instead of unpacking the dictionary here, could we do
```suggestion
    data_assistant_result: DataAssistantResult = context.assistants.volume.run(
        batch_request=batch_request,
        exclude_column_names=exclude_column_names
    )
```

to make the functionality more apparent?

Reply:
@talagluck I think I can try to make a go of it and see if the arguments can be gathered downstream.  I will let you know.